### PR TITLE
[strings] Rename Macedonia to North Macedonia

### PR DESCRIPTION
**Pull request by alnzrv**
The country changed its name in 2019.

### CODE REVIEW

File: data/countries_names.txt [hunk 1]
```diff
@@ -23308,41 +23308,41 @@
...
+  nl = Noord-Macedonië
+  pl = Macedonia Północna
+  pt = República da Macedónia do Norte
+  ro = Republica Macedonia de Nord
+  ru = Республика Северная Македония
+  sk = Severné Macedónsko
+  sv = Nordmakedonien
+  th = ประเทศมาซิโดเนียเหนือ
+  tr = Kuzey Makedonya Cumhuriyeti
+  uk = Республіка Північна Македонія
```
Comment (biodranik) [inline data/countries_names.txt:23342]:
```suggestion
  uk = Північна Македонія
```

File: data/countries_names.txt [hunk 2]
```diff
@@ -23308,41 +23308,41 @@
...
+  it = Repubblica della Macedonia del Nord
+  ja = 北マケドニア共和国
+  ko = 북마케도니아 공화국
+  mr = उत्तर मॅसेडोनिया
+  nb = Republikken Nord-Makedonia
+  nl = Noord-Macedonië
+  pl = Macedonia Północna
+  pt = República da Macedónia do Norte
+  ro = Republica Macedonia de Nord
+  ru = Республика Северная Македония
```
Comment (biodranik) [inline data/countries_names.txt:23337]:
```suggestion
  ru = Северная Македония
```

File: data/countries_names.txt [hunk 3]
```diff
@@ -23308,41 +23308,41 @@
...
+  id = Republik Makedonia Utara
+  it = Repubblica della Macedonia del Nord
+  ja = 北マケドニア共和国
+  ko = 북마케도니아 공화국
+  mr = उत्तर मॅसेडोनिया
+  nb = Republikken Nord-Makedonia
+  nl = Noord-Macedonië
+  pl = Macedonia Północna
+  pt = República da Macedónia do Norte
+  ro = Republica Macedonia de Nord
```
Comment (biodranik) [inline data/countries_names.txt:23336]:
```suggestion
  ro = Macedonia de Nord
```

File: data/countries_names.txt [hunk 4]
```diff
@@ -23308,41 +23308,41 @@
...
+  ru = Республика Северная Македония
+  sk = Severné Macedónsko
+  sv = Nordmakedonien
+  th = ประเทศมาซิโดเนียเหนือ
+  tr = Kuzey Makedonya Cumhuriyeti
```
Comment (biodranik) [inline data/countries_names.txt:23341]:
```suggestion
  tr = Kuzey Makedonya
```

File: data/countries_names.txt [hunk 5]
```diff
@@ -23308,41 +23308,41 @@
...
+  fi = Pohjois-Makedonia
+  fr = République de Macédoine du Nord
+  he = מקדוניה הצפונית
+  hu = Észak-Macedónia
+  id = Republik Makedonia Utara
+  it = Repubblica della Macedonia del Nord
+  ja = 北マケドニア共和国
+  ko = 북마케도니아 공화국
+  mr = उत्तर मॅसेडोनिया
+  nb = Republikken Nord-Makedonia
```
Comment (biodranik) [inline data/countries_names.txt:23332]:
```suggestion
  nb = Nord-Makedonia
```

File: data/countries_names.txt [hunk 6]
```diff
@@ -23308,41 +23308,41 @@
...
+  el = Βόρεια Μακεδονία
+  es = República de Macedonia del Norte
+  et = Põhja-Makedoonia
+  eu = Ipar Mazedoniako Errepublika
+  fi = Pohjois-Makedonia
+  fr = République de Macédoine du Nord
+  he = מקדוניה הצפונית
+  hu = Észak-Macedónia
+  id = Republik Makedonia Utara
+  it = Repubblica della Macedonia del Nord
```
Comment (biodranik) [inline data/countries_names.txt:23328]:
```suggestion
  id = Makedonia Utara
  it = Macedonia del Nord
```

File: data/countries_names.txt [hunk 7]
```diff
@@ -23308,41 +23308,41 @@
...
+  ca = Macedònia del Nord
+  cs = Severní Makedonie
+  da = Nordmakedonien
+  de = Nordmazedonien
+  el = Βόρεια Μακεδονία
+  es = República de Macedonia del Norte
+  et = Põhja-Makedoonia
+  eu = Ipar Mazedoniako Errepublika
+  fi = Pohjois-Makedonia
+  fr = République de Macédoine du Nord
```
Comment (biodranik) [inline data/countries_names.txt:23324]:
```suggestion
  fr = Macédoine du Nord
```

File: data/countries_names.txt [hunk 8]
```diff
@@ -23308,41 +23308,41 @@
...
-  uk = Республіка Македонія
-  vi = Cộng hòa Macedonia
-  zh-Hans = 马其顿共和国
-  zh-Hant = 馬其頓共和國
+  ca = Macedònia del Nord
+  cs = Severní Makedonie
+  da = Nordmakedonien
+  de = Nordmazedonien
+  el = Βόρεια Μακεδονία
+  es = República de Macedonia del Norte
```
Comment (biodranik) [inline data/countries_names.txt:23320]:
```suggestion
  es = Macedonia del Norte
```

**Comment by RedAuburn** [comment]
Did the borders change, or was it just a renaming?

**Review by strump — APPROVED** [review]

**Pull request merged**